CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ All notable changes to this project will be documented in this file.
 * ### `nidcpower` (NI-DCPower)
     * #### Added
     * #### Changed
+        * The byte packing alignment of `struct_NILCRLoadCompensationSpot` is changed from 4-byte to 8-byte in Linux.
```

Comment:
Do we have a special way for calling out binary breakers in the changelog?

Reply:
Not really because they are soooo unusual.

I actually don't like how the change is worded because it references an internal implementation detail. That's meaningless to customers, they don't interact with `struct_NILCRLoadCompensationSpot`.

I think we need to call out the compatibility breakage here in terms customers care.

> Binary compatibility change for type `LCRLoadCompensationSpot ` on Linux. Client code using method `nidcpower.Session.perform_lcr_load_compensation` on Linux now requires NI-DCPower 2023 Q1 driver runtime or newer.

I think think this is enough. It gives customers information that is truthful, useful, and actionable to them rather than just paraphrasing a diff which a Python user won't know what to do with.

You guys like it?

Reply:
Yes, I think that sounds good. I have updated the CHANGELOG in my latest commit [6d0b3fb](https://github.com/ni/nimi-python/pull/1870/commits/6d0b3fba52ee8295e821088da8812f0da19eddec).